### ISSUE TITLE
Audio drops when using keyframes in the clip

### ISSUE THREAD

**Issue by WaqasAwan00** [OpenShot/openshot-qt#2408]
After importing an audio clip, using a set of keyframes will cause the audio to drop 1 minute after the clip starts. (I hope i know what keyframe means here but if not I will explain): 

Windows 10
 - OpenShot Ver [2.4.3]

**To Reproduce**
For example say the entire audio clip is at a volume of 0.50, I move the marker/flag to the middle, and set the volume to 0.40, and go to the end of the clip and set the volume to 0.00, so it effectively gets softer from 0.40 to 0.00 as the clip plays. After exporting, the audio will just drop entirely after 1 minute of playback, regardless of where the keyframe (volume specification time) actually is. 

**Expected behavior**
Audio should develop normally: 0.50 --> 0.40 -->0.00, in the exported clip.
So far to work around this, I've stopped editing audio and when exporting the video all the audio plays normally. But of course, this renders audio editing unusable.

So to summarize: editing audio volumes in an audio clip causes the audio to drop after 1 minute in the exported video. This is NOT present in the preview in-editor, it just plays normally in the editor.

**Comment by WaqasAwan00** [comment]
Yes! I couldn't quite get the hang of the auto- fade feature it had that put in a fade through the right click options (volume would jump right before the actual fade).
So i thought to custom adjust the volume but ended up with the problem specified.

**Comment by speedytechdev** [comment]
Can you take a screenshot of your timeline and send it to me, please.

**Comment by WaqasAwan00** [comment]
Here is my timeline: 
![image](https://user-images.githubusercontent.com/34248727/49271684-b663a980-f43c-11e8-8d2e-26dff2b72f29.png)

**Comment by speedytechdev** [comment]
That looks normal could you send me a copy of your film, please.

**Comment by WaqasAwan00** [comment]
Will do, just gotta re-export; been deleting the ones with cut-out audio. Just a sec

**Comment by speedytechdev** [comment]
Ok, I am positive we have had this issue before I just can't remember how we fixed it.

**Comment by WaqasAwan00** [comment]
What formats should I upload, btw? I imagine you'd want the file to open in an editor yourself too

**Comment by speedytechdev** [comment]
Mp4 please if possible and yes please

**Comment by WaqasAwan00** [comment]
I can't seem to replicate it on a smaller video, maybe I'm just messing something up? Also I'm having a bit of trouble uploading video files here. If I encounter this problem again, I'll make an upload to youtube and put a link here.

On a side note, trying to open a project whose base files were deleted puts the program into a loop where it keeps asking to identify the folder where the project is located despite not actually being able to open it. Just a side-bug i found.

**Comment by speedytechdev** [comment]
ok, that sounds good. Yep, we know about that. It is fixed in a daily build. When the next main version build comes out it should include that fix but thanks.

**Comment by speedytechdev** [comment]
@WaqasAwan00 This issue is going to be closed is that ok with you?

**Comment by WaqasAwan00** [comment]
yeah that's fine, ill open a ticket when/if i can find that bug again
